src/components/ActivityPage/Transaction/TransactionDetail/TransactionDetail.tsx

```diff
@@ -14,9 +14,9 @@ import './TransactionDetail.css'
 
 const getHref = (tx: Transaction) => {
   if (tx.status === null) {
-    return
+    return ''
   }
-  return getEtherscanHref({ txHash: tx.replacedBy || tx.hash })
+  return getTransactionHref({ txHash: tx.replacedBy || tx.hash })
```

Comment:
What do you think of creating an issue to move this to `dcl-dapps` directly?

Reply:
You mean adding a method that takes a Transaction and returns an `href`? I think it's a good idea but I'm not sure how specific to the builder this method is. I'll add the issue there and ask juanca, linking this comment